docs/topics/con_configmap-availability.adoc

```diff
@@ -0,0 +1,145 @@
+[id='using_a_configmap_{context}']
+= Using a ConfigMap
+
+:deploymentPlatform: Kubernetes
+
+== ConfigMap PropertySource
+
+
+//openshift docs on configmaps
+//https://docs.openshift.org/latest/dev_guide/configmaps.html
+
+
+//replace Kubernetes with openshift?
+{deploymentPlatform} provides a resource named link:http://kubernetes.io/docs/user-guide/configmap/[`ConfigMap`] to externalize the
+parameters to pass to your application in the form of key-value pairs or embedded `application.properties|yaml` files.
+Your application automatically redeploys when the ConfigMap changes.
+`ConfigMapPropertySource` searches the deployment configuration of your application for a Kubernetes `ConfigMap` object in which the `metadata.name` property matches the name of your {SpringBoot} application (set by the `spring.application.name` property).
```

Comment:
I feel we're missing some context here but maybe it's because I'm not seeing the complete rendered doc? We should explain that this comes from Spring Cloud Kubernetes and point to the doc?